bitrise.yml

```diff
@@ -231,6 +231,39 @@ workflows:
     - opts:
         is_expand: false
       INTEGRATOR_VARIANT: debug
+  upgrade_dependencies:
+    steps:
+    - activate-ssh-key@4: {}
+    - git-clone@6: {}
+    - cache-pull@2: {}
+    - install-missing-android-tools@3:
+        inputs:
+        - gradlew_path: $PROJECT_LOCATION/gradlew
+    - gradle-runner@2:
+        inputs:
+        - gradlew_path: $PROJECT_LOCATION/gradlew
+        - gradle_file: $PROJECT_LOCATION/build.gradle
+        - gradle_task: saveCoreSdkVersion --coreSdkVersion=$NEW_VERSION
+    - script@1:
```

Comment:
Does this step do the same thing as 'scripts/create-pr-for-increment-project-version.sh'?
In this case, maybe you can just run that script not to duplicate logic?